### DISCUSSION TITLE
Dashboards Reverting

### DISCUSSION THREAD

**Discussion by EchoGangster** [Security-Onion-Solutions/securityonion#4387] [Unsupported Versions] [answered]
Found out today that if you modify any of the prebuilt visualizations and dashboards that they will revert to original state. I saw in the documentation that there is a way to revert dashboard but would like to be able to customize the prebuilt ones to fit organizations needs.

**Comment by dougburks** [comment, marked as answer]
Instead of modifying the prebuilt dashboards, copy the desired dashboard and then edit the copy.
